torchinductor/codegen/wrapper.py

```diff
@@ -24,10 +25,15 @@
 
 
 def buffer_reuse_key(node: ir.Buffer):
```

Comment:
Should we consider is_contiguous in the key as well?

Reply:
This is for reusing memory, so layout of the freed buffer doesn't need to match the layout of what we reuse it for.